src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Reissuance/ReissuanceConsent/HealthCertificateReissuanceConsentViewModel.swift

```diff
@@ -223,9 +226,18 @@ final class HealthCertificateReissuanceConsentViewModel {
 					self.submit(
 						with: resource,
 						requestCertificates: requestCertificates,
+						dispatchGroup: dispatchGroup,
 						completion: completion
```

Comment:
I think it would be better to not pass in the dispatch group and distribute its calls in multiple functions, but use the completion closure to call leave on the dispatch group and then call the injected completion.

Reply:
Okay, have to agree with @nickguendling here I completely missed the `leave()` because of this.

Reply:
@marcussc  @nickguendling  done